src/sentry/tasks/reports.py

```diff
@@ -393,21 +393,53 @@ def build_message(timestamp, duration, organization, user, report):
     return message
 
 
+DISABLED_ORGANIZATIONS_USER_OPTION_KEY = 'reports:disabled-organizations'
+
+
+def user_subscribed_to_organization_reports(user, organization):
+    return organization.id not in set(
+        UserOption.objects.get_value(
+            user=user,
+            project=None,
+            key=DISABLED_ORGANIZATIONS_USER_OPTION_KEY,
+            default=[],
+        )
+    )
+
+
+class Skipped(object):
+    NotSubscribed = object()
+    NoProjects = object()
+
+
 @instrumented_task(
     name='sentry.tasks.reports.deliver_organization_user_report',
     queue='reports.deliver')
 def deliver_organization_user_report(timestamp, duration, organization_id, user_id):
     organization = _get_organization_queryset().get(id=organization_id)
     user = User.objects.get(id=user_id)
 
+    if not user_subscribed_to_organization_reports(user, organization):
+        logger.debug(
+            'Skipping report for %r to %r, user is not subscribed to reports.',
```

Comment:
Debug is ok to pretty much do whatever we want, seeing as how it's meant for human eyes only.